### PR TITLE
Allow Symfony 6

### PR DESCRIPTION
**Pull request by Kocal**
Some tests are failing, but that was already the case before this PR (eg: in https://github.com/sensiolabs/SensioFrameworkExtraBundle/runs/3084739718?check_suite_focus=true).
Unfortunately I don't know how to fix them.

Thanks!

### CODE REVIEW

File: tests/Fixtures/config/config_sf5.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+security:
+  password_hashers:
+    Symfony\Component\Security\Core\User\User: plaintext
```
Comment (derrabus) [inline tests/Fixtures/config/config_sf5.yml:3]: Please use four spaces for indentation

Comment (Kocal) [reply tests/Fixtures/config/config_sf5.yml:3]: Fixed, thanks!

Comment (Kocal) [reply tests/Fixtures/config/config_sf5.yml:3]: Fixed, thanks!

**Review by Nyholm — APPROVED** [review]
Thank you. 

I agree with this change. I would be happy to see it merged.

**Comment by fabpot** [comment]
Thank you @Kocal.

**Pull request merged**